### PR TITLE
fix:  add chart icon and fix some chart bugs

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

The chart required some color adjustments.
Additionally, there was an issue with the tooltip's display and closing because it was opened in a modal window. This issue has been resolved in this PR, and this PR now has a chart icon.

### CODE REVIEW

File: widget/charts/src/components/BarChart/BarChart.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ export const BarChart = (props: BarChartPropTypes) => {
     buckets,
     margin = DEFAULT_MARGIN,
     getLabel,
+    isDarkTheme = false,
```
Comment (nikaaru) [inline widget/charts/src/components/BarChart/BarChart.tsx:63]: Can't we handle the dark theme inside the chart itself?

File: widget/ui/package.json [hunk 2]
```diff
@@ -26,7 +26,6 @@
   },
   "peerDependencies": {
     "@lingui/core": "4.2.1",
-    "@lingui/react": "4.2.1",
```
Comment (nikaaru) [inline widget/ui/package.json:29]: What was the need to remove `@lingui/react` from `peerDependencies` and `devDependencies`
And we put it in `dependencies`?

Comment (mikasackermn) [reply widget/ui/package.json]: fixed it

File: widget/charts/src/components/BarChart/BarChart.tsx (hunk 1 above)
Comment (mikasackermn) [reply widget/charts/src/components/BarChart/BarChart.tsx:63]: I test this but I didn't find any solution to handle it.

**Review by nikaaru — APPROVED** [review]
LGTM

**Comment by nikaaru** [comment]
Please improve the commit text
For example:

fix: add chart icon and handle dark theme in BarChart component

**Pull request merged**